src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/PostgresAccessPolicyDAO.scala

```diff
@@ -67,7 +67,10 @@ class PostgresAccessPolicyDAO(protected val dbRef: DbReference,
         .update().apply()
     } else {
       samsql"""delete from ${RoleActionTable as ra}
-                 where (${ra.resourceRoleId}, ${ra.resourceActionId}) not in (${roleActionValues})"""
+                 using ${ResourceRoleTable as rr}
+                 where ${ra.resourceRoleId} = ${rr.id}
+                 and (${ra.resourceRoleId}, ${ra.resourceActionId}) not in (${roleActionValues})
+                 and ${rr.resourceTypeId} = ${resourceTypePK}"""
```

Comment:
Unrelated bug undiscovered because our code did not use the RoleActionTable. But now it does so this is important. Without this change every saved resource type wipes out all the RoleActionTable entries for any other resource type.